doc/source/cluster/metrics.md

```diff
@@ -214,7 +218,8 @@ If this is your first time using Grafana, login with the username: `admin` and p
 
 ![grafana login](images/graphs.png)
 
-**Troubleshooting: Using Ray configurations in Grafana with Homebrew on macOS X**
```

Comment:
Thanks for merging master again, it looks like the same test failed again.  It's probably due to some other issue in the file, but I have no idea what it is and we can't merge the PR with the test failing.  Because the error involves "header", I'm going to suggest using `** **` instead of `#` headers on the changed lines as a workaround, just to see if the error goes away so we can unblock this PR... do you mind giving it a shot?

Reply:
@architkulkarni this fixed the issue ! There are stil some other tests failing though , one looks like a timeout issue and other is validating url redirections. I think they are unrelated to this commit. Can you check and confirm once as well?